users/serializers.py

```diff
@@ -0,0 +1,27 @@
+from rest_framework import serializers
+from django.contrib.auth.password_validation import validate_password
+
+from .models import User
+
+
+class UserSerializer(serializers.ModelSerializer):
```

Comment:
in the serializer, you should add a method to validate that password1 == password2 split from the create method.